src/settings-ui/Microsoft.PowerToys.Settings.UI/Microsoft.PowerToys.Settings.UI.csproj

```diff
@@ -275,7 +286,7 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
-    <PRIResource Include="Strings\*\Resources.resw" />
+    <PRIResource Include="Strings\en-us\Resources.resw" />
```

Comment:
this diff raises an eye brow

Reply:
Hm... I don't think I explicitly made it. Should I reverse this change? @crutkas

Reply:
This one yes

Reply:
@crutkas this should be back to asterisk.